tests/test.linter.js

```diff
@@ -549,7 +549,9 @@ describe('Linter.getAddonMetadata()', function() {
       });
   });
 
-  it('should recognise it as a Jetpack add-on', () => {
+  it('should recognise it as a Jetpack add-on', function() {
+    this.timeout(20000);
```

Comment:
I’ll check, but I know running the dispensary on add ons with lots of JS files slows things down, but hopefully not that much.